### ISSUE TITLE
Can pirouette still run on Peregrine?

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/peregrine#9]
There are [too](https://github.com/richelbilderbeek/razzo_project/issues/3) [many](https://github.com/richelbilderbeek/raket/issues/43) [Issues](https://github.com/richelbilderbeek/razzo/issues/251) that hint a no.

**Comment by richelbilderbeek** [comment]
This one works:

```
p230198@peregrine:peregrine cat scripts/run_pir.R 
# R script to test if pirouette runs on peregrine
pirouette::pir_run(
  phylogeny = ape::read.tree(text = "((A:2, B:2):1, C:3);"),
  pir_params = peregrine::create_test_pff_pir_params()
)
```

```
p230198@peregrine:peregrine sbatch scripts/run_r_script scripts/run_pir.R 

Submitted batch job 7443244

             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
           7443244   regular run_r_sc  p230198 CG       0:16      1 pg-node027

p230198@peregrine:peregrine cat *.log

The following have been reloaded with a version change:
  1) GCCcore/7.3.0 => GCCcore/6.4.0
  2) zlib/1.2.11-GCCcore-7.3.0 => zlib/1.2.11-GCCcore-6.4.0


The following have been reloaded with a version change:
  1) Ghostscript/9.23-GCCcore-6.4.0 => Ghostscript/9.22-GCCcore-6.4.0-cairo-1.14.12
  2) ImageMagick/6.9.10-5-GCCcore-6.4.0 => ImageMagick/7.0.7-26-foss-2018a
  3) LittleCMS/2.9-GCCcore-6.4.0 => LittleCMS/2.9-foss-2018a
  4) expat/2.2.5-GCCcore-7.3.0 => expat/2.2.5-GCCcore-6.4.0

Rscript scripts/run_pir.R
Registered S3 method overwritten by 'geiger':
  method            from
  unique.multiPhylo ape 
  tree inference_model inference_model_weight site_model clock_model tree_prior
1 true      generative                     NA       JC69      strict       yule
     error_1 error_2   error_3
1 0.04450728 0.19498 0.1912832


###############################################################################
Peregrine Cluster
Job 7443244 for user 'p230198'
Finished at: Tue Aug 27 12:23:58 CEST 2019

Job details:
============

Name                : run_r_script
User                : p230198
Partition           : regular
Nodes               : pg-node027
Cores               : 1
State               : COMPLETED
Submit              : 2019-08-27T12:23:42
Start               : 2019-08-27T12:23:42
End                 : 2019-08-27T12:23:58
Reserved walltime   : 10:00:00
Used walltime       : 00:00:16
Used CPU time       : 00:00:11 (efficiency: 74.70%)
% User (Computation): 87.02%
% System (I/O)      : 12.97%
Mem reserved        : 10G/node
Max Mem used        : 0.00  (pg-node027)
Max Disk Write      : 0.00  (pg-node027)
Max Disk Read       : 0.00  (pg-node027)
```

**Comment by richelbilderbeek** [comment]
Gotcha!

```
p230198@peregrine:peregrine sbatch scripts/run_r_script scripts/run_pir.R 
Submitted batch job 7443269

p230198@peregrine:peregrine q
             JOBID PARTITION     NAME     USER ST       TIME  NODES NODELIST(REASON)
           7443269   regular run_r_sc  p230198  R       0:11      1 pg-node009

p230198@peregrine:peregrine cat run_r_script_7443269.log 

The following have been reloaded with a version change:
  1) GCCcore/7.3.0 => GCCcore/6.4.0
  2) zlib/1.2.11-GCCcore-7.3.0 => zlib/1.2.11-GCCcore-6.4.0


The following have been reloaded with a version change:
  1) Ghostscript/9.23-GCCcore-6.4.0 => Ghostscript/9.22-GCCcore-6.4.0-cairo-1.14.12
  2) ImageMagick/6.9.10-5-GCCcore-6.4.0 => ImageMagick/7.0.7-26-foss-2018a
  3) LittleCMS/2.9-GCCcore-6.4.0 => LittleCMS/2.9-foss-2018a
  4) expat/2.2.5-GCCcore-7.3.0 => expat/2.2.5-GCCcore-6.4.0

Rscript scripts/run_pir.R
Registered S3 method overwritten by 'geiger':
  method            from
  unique.multiPhylo ape 
Error in value[[3L]](cond) : Could not estimate the marginal likelihood. 
Error message: File 'output_log_filename' not found. Could not find file with path '/local/tmp/Rtmpoc5Ir4/file17ada2d07488.log'
site_model$name: GTR
clock_model$name: relaxed_log_normal
tree_prior$name: birth_death
epsilon: 1e-12
rng_seed: NA
verbose: FALSE
beast2_working_dir: /home/p230198/.cache/file17ada73b9da98/beast2_tmp_folder17ada5721a71f
beast2_bin_path: /home/p230198/.local/share/beast/bin/beast
Calls: <Anonymous> ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
In addition: Warning messages:
1: In file.rename(from = actual_log_filename, to = output_log_filename) :
  cannot rename file '/home/p230198/.cache/file17ada73b9da98/beast2_tmp_folder17ada5721a71f/pff_17adac1a0d9b_evidence.log' to '/local/tmp/Rtmpoc5Ir4/file17ada2d07488.log', reason 'Invalid cross-device link'
2: In file.rename(from = actual_trees_filename, to = to) :
  cannot rename file '/home/p230198/.cache/file17ada73b9da98/beast2_tmp_folder17ada5721a71f/pff_17adac1a0d9b_evidence.trees' to '/local/tmp/Rtmpoc5Ir4/file17ada7c47b2c2.trees', reason 'Invalid cross-device link'
Execution halted


###############################################################################
Peregrine Cluster
Job 7443269 for user 'p230198'
Finished at: Tue Aug 27 12:35:23 CEST 2019

Job details:
============

Name                : run_r_script
User                : p230198
Partition           : regular
Nodes               : pg-node009
Cores               : 1
State               : FAILED
Submit              : 2019-08-27T12:34:37
Start               : 2019-08-27T12:34:38
End                 : 2019-08-27T12:35:23
Reserved walltime   : 10:00:00
Used walltime       : 00:00:45
Used CPU time       : 00:00:41 (efficiency: 93.16%)
% User (Computation): 92.83%
% System (I/O)      :  7.17%
Mem reserved        : 10G/node
Max Mem used        : 330.92M (pg-node009)
Max Disk Write      : 133.12K (pg-node009)
Max Disk Read       : 3.14M (pg-node009)
```

**Comment by richelbilderbeek** [comment]
The error is in the model comparison, check upstream at [mcbette Issue 12](https://github.com/richelbilderbeek/mcbette/issues/12)

**Comment by richelbilderbeek** [comment]
Bug was in mcbette, fixed with mcbette v1.6.
